bpf/bpf_lxc.c

```diff
@@ -269,6 +269,12 @@ static __always_inline int ipv6_l3_from_lxc(struct __ctx_buff *ctx,
 #ifdef ENABLE_NODEPORT
 		/* See comment in handle_ipv4_from_lxc(). */
 		if (ct_state.node_port) {
+#ifdef NETFILTER_COMPAT_MODE
+			/* Pass the packet to the stack and let bpf_host perform
+			 * reverse NAT at egress of the native device.
+			 */
+			return CTX_ACT_OK;
```

Comment:
Just to double check, will it have the correct security identity for the host-fw? (/cc @pchaigno)

Reply:
Ah, that's a good point :thinking: 

If reverse NAT is now performed after the enforcement of host policies, I assume it will break enforcement... Couldn't we move the reverse NAT in `bpf_host` to before the `ENABLE_HOST_FIREWALL` block?

Reply:
Hi @pchaigno can you help me to understand, what information is lost ? if reverse DNAT performed after `ENABLE_HOST_FIREWALL` block.

Reply:
Ah! No, you're right! That should be alright. The host firewall will see DNATed, backend addresses on the way forward (because we DNAT first) and also DNATed, backend addresses on the way back (because we didn't run reverse DNAT yet). @brb Did you have any other concern?